arcade/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "arcade-ai"
-version = "1.0.1"
+version = "1.1.0"
```

Comment:
Since most people installed as `pip install 'arcade-ai[fastapi]'` I think it might be okay to only make this a minor release.